### PR TITLE
Add Keychron Q60

### PR DESCRIPTION
**Pull request by lalalademaxiya1**
We added new keyboard Q60.
Thanks!

## Description

We added ANSI versions.
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keychron/q60/config.h [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 14
+
```
Comment (waffle87) [inline keyboards/keychron/q60/config.h:22]:
```suggestion
```

File: keyboards/keychron/q60/q60.c [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "q60.h"
```
Comment (waffle87) [inline keyboards/keychron/q60/q60.c:17]:
```suggestion
#include "quantum.h"
```

File: keyboards/keychron/q60/q60.h [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/* Copyright 2022 @ Keychron (https://www.keychron.com)
```
Comment (waffle87) [inline keyboards/keychron/q60/q60.h:1]: This file can be removed.

**Comment by keychron-dev** [comment]
@waffle87 please have a look at this too

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Pull request merged**